examples/siro_sandbox/sandbox_app.py

```diff
@@ -249,7 +273,9 @@ def viz_and_get_grasp_drop_hints(self):
                         )
                         if self.gui_input.get_key_down(GuiInput.KeyNS.SPACE):
                             grasp_object_id = hit_info.object_id
-                            self._held_target_obj_idx = 0  # temp hack; no way to look this up currently from hit_info.object_id
+                            self._held_target_obj_idx = (
```

Comment:
Use correct object index.

Reply:
Did you test on an episode with multiple target objects?

Reply:
Yes, I've tested on FP scenes with 2 target objects to rearrange. Works as expected.